### ISSUE TITLE
Add NATS project

### ISSUE THREAD

**Issue by lukaszgryglicki** [cncf/devstats.archive#80]
Add NATS project:
- https://github.com/nats-io
- https://nats.io/
- Add on the test servers first
- Try to automate as much as possible on the way
- Add on the prod after this

**Comment by lukaszgryglicki** [comment]
Will work on this tomorrow, ETA 2-3 days.

**Comment by dankohn** [comment]
NATS logo: https://github.com/cncf/artwork/pull/38/files?short_path=df09cdb#diff-df09cdbab4a7b2271dfb961f61918b10

Could you please consider switching all of the logos from PNGs to SVGs to get higher resolution with a lower file size. No worries if you want to hold off until later.

**Comment by lukaszgryglicki** [comment]
Thanks, some logos use PNG, some other use SVG.
But I have no control over this.
It is a branded Grafana.
I can **only** switch logos on the home dashboards (those pointing to all other CNCF projects).
And yes, I'll postpone this a big because I have a lot of stuff very wip atm.

**Comment by dankohn** [comment]
Please create a new issue. Because all CNCF logos are available as SVG at
https://github.com/cncf/artwork/

--
Dan Kohn <dan@linuxfoundation.org>
Executive Director, Cloud Native Computing Foundation https://www.cncf.io
+1-415-233-1000 https://www.dankohn.com

On Mon, Mar 12, 2018 at 10:15 AM, Łukasz Gryglicki <notifications@github.com

**Comment by lukaszgryglicki** [comment]
Sorry, I don't understand... Issue on Grafana to support SVG everywhere? 
Or issue on devstats to use SVG icons on the home dashboard?
If the latter, then I'll just do it soon - probably tomorrow.

**Comment by dankohn** [comment]
Issue on DevStats to switch CNCF logos from PNGs to SVGs.

--
Dan Kohn <dan@linuxfoundation.org>
Executive Director, Cloud Native Computing Foundation https://www.cncf.io
+1-415-233-1000 https://www.dankohn.com

On Mon, Mar 12, 2018 at 10:28 AM, Łukasz Gryglicki <notifications@github.com

**Comment by lukaszgryglicki** [comment]
OK, will do.

**Comment by lukaszgryglicki** [comment]
https://github.com/cncf/devstats/issues/82

**Comment by lukaszgryglicki** [comment]
NATS is done: https://nats.devstats.cncf.io/d/8/dashboards?refresh=15m&orgId=1
And in "All CNCF" too: https://all.devstats.cncf.io/d/8/dashboards?refresh=15m&orgId=1
Icons from cncf/artwork PR: https://github.com/cncf/artwork/pull/38

**Comment by dankohn** [comment]
Great, but next time, please don't merge into production until the vote is
marked as complete.

--
Dan Kohn <dan@linuxfoundation.org>
Executive Director, Cloud Native Computing Foundation https://www.cncf.io
+1-415-233-1000 https://www.dankohn.com

**Comment by lukaszgryglicki** [comment]
OK
